pkgs/data/misc/brise/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, librime }:
+
+stdenv.mkDerivation rec {
+  name = "brise-0.35.20170916";
+
+  src = fetchFromGitHub {
+    owner = "rime";
+    repo = "brise";
+    rev = "1cfb0fe1d3a4190ce5d034f141941156dd271e80";
+    sha256 = "1l13j3cfwida0ycl874fizz2jwjvlxid589a1iciqa9y25k21ql7";
+  };
+
+  prePatch = ''
+    patchShebangs scripts/*
+  '';
+
+  # we need to use fetchFromGitHub to fetch sub-packages before we 'make',
+  # since nix won't allow networking during 'make'
+  preBuild = import ./fetchPackages.nix fetchFromGitHub;
+
+  postBuild = ''
+    # also build binary files
+    BRISE_BUILD_BINARIES=yes make
+  '';
+
+  installPhase = ''
+    make install PREFIX=$out
+  '';
+
+  buildInputs = [ librime ];
+
+  meta = with stdenv.lib; {
+    description = "Rime Schema Repository";
+    longDescription = ''
+      This software is a collection of data packages used by Rime
+      to support various Chinese input methods, including those based on
+      modern dialects or historical diasystems of the Chinese language.
+    '';
+    homepage = http://rime.im;
+    # Note that individual packages in this collection
+    # may be released under different licenses
+    license = licenses.gpl3;
+    platforms = platforms.all;
+    maintainers = [ maintainers.mingchuan ];
```

Comment:
While you wait, you should probably update this to match your new username (and same in the other files and lib/maintainers.nix). :)

Reply:
Hmm, `sifmelcara` is my GitHub handle, and `mingchuan` is the name I used in git commit messages.
However, I can change my name in lib/maintainers.nix so that people can easily ping me on GitHub.

Reply:
Oh right, my bad. Yeah at the top of lib/maintainers.nix it says github handle is preferred, thanks. :sparkles: